invenio/b2share/modules/b2deposit/restful.py

```diff
@@ -66,6 +68,7 @@
     'domain':               ('980__a', False), #\ same marctag
     'resource_type':        ('980__a', True),  #/ same marctag
     'publication_date':     ('260__c', False),
+    'publisher':            ('8560_f', False),
```

Comment:
This is not the publisher field. The publisher field is 260/b, and in our case always the url of b2share. What are we trying to do here?